### PR TITLE
[9.0] [MIG] base_location_geonames_import

### PR DESCRIPTION
**Pull request by pedrobaeza**
- Make module installable
- Short headers
- Update README.rst
- Improve test efficiency

### CODE REVIEW

**Comment by bealdav** [comment]
FYI, yml will be deprecated

https://www.odoo.com/fr_FR/groups/community-59/community-14056914

**Comment by pedrobaeza** [comment]
Thanks for the info, and I prefer Python files, but for this module, I take the easy path of only update existing YAML file. As in v9 is still supported, I prefer to let as is.

File: base_location_geonames_import/wizard/geonames_import.py [hunk 1]
```diff
@@ -1,29 +1,11 @@
...
-#
-##############################################################################
+# © 2014 Alexis de Lattre <alexis.delattre@akretion.com>
+# © 2014 Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+# © 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from openerp import models, fields, api, _
-from openerp.exceptions import Warning
+from openerp.exceptions import Warning as UserError
```
Comment (lmignon) [inline base_location_geonames_import/wizard/geonames_import.py]: Why do you have aliased 'Warning'?

Comment (pedrobaeza) [inline base_location_geonames_import/wizard/geonames_import.py]: This is a guideline because Warning is deprecated

Comment (pedrobaeza) [inline base_location_geonames_import/wizard/geonames_import.py]: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms

Comment (pedrobaeza) [inline base_location_geonames_import/wizard/geonames_import.py]: I'm seeing that in v9 I should import directly UserError

Comment (pedrobaeza) [inline base_location_geonames_import/wizard/geonames_import.py]: @lmignon, changed to use directly `UserError`

**Comment by lmignon** [comment]
Thank you @pedrobaeza the code looks good to me. :+1: once Travis is green.

**Comment by pedrobaeza** [comment]
@bealdav, with the advices from @alexis-via in https://github.com/OCA/partner-contact/pull/254, I have converted tests to Python, and improve a bit more the efficiency.

**Comment by simahawk** [comment]
:+1:

**Comment by pedrobaeza** [comment]
We can merge this one.

**Pull request merged**